infra/src/feature-env.ts

```diff
@@ -98,9 +99,21 @@ const buildIngressComment = (data: HelmService[]): string =>
     .join('\n')
 
 const buildComment = (data: Services<HelmService>): string => {
-  return `Feature deployment successful! Access your feature here:\n\n${buildIngressComment(
-    Object.values(data),
-  )}`
+  if (Object.values(data.ingress).length > 0) {
+    return `Feature deployment of your services will begin shortly. Your feature will be accessible here:\n\n${buildIngressComment(
+      Object.values(data),
+    )}`
+  } else {
+    return `Feature deployment of your services will begin shortly. No web endpoints defined (no ingresses were defined)`
+  }
+}
```

Comment:
What about adding a link to DD in this text? Such as `https://app.datadoghq.eu/logs?query=kube_namespace%{KUBE_NAMESPACE}`